content/en/docs/usage/gateway-api.md

```diff
@@ -0,0 +1,347 @@
+    - hostname: *.example.com
+      port: 443
+      protocol: HTTPS
+      routes:
+        kind: HTTPRoute
+        selector:
+          matchLabels:
+            app: foo
+      tls:
+        mode: Terminate
+        certificateRef:
+          name: example-com-tls
+          kind: Secret
+          group: core
+
+    # Listener 3: also same Secret name, except the hostname is also the same.
+    - hostname: *.example.com
+      port: 8443
+      protocol: HTTPS
+      routes:
+        kind: HTTPRoute
+        selector:
+          matchLabels:
+            app: foo
+      tls:
+        mode: Terminate
+        certificateRef:
+          name: example-com-tls
+          kind: Secret
+          group: core
+
+   # Listener 4: different Secret name.
+    - hostname: site.org
+      port: 443
+      protocol: HTTPS
+      routes:
+        kind: HTTPRoute
+        selector:
+          matchLabels:
+            app: foo
+      tls:
+        mode: Terminate
+        certificateRef:
+          name: site-org-tls
+          kind: Secret
+          group: core
+```
+
+cert-manager will create two Certificates since two Secret names are used:
+`example-com-tls` and `site-org-tls`. Note the Certificate's `dnsNames` contains
+a single occurrence of `*.example.com ` for both listener 2 and 3 (the
+`hostname` values are de-duplicated).
+
+The two Certificates look like this:
+
+```yaml
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: example-com-tls
+spec:
+  issuerRef:
+    name: my-issuer
+    kind: Issuer
+    group: cert-manager.io
+  dnsNames:
+    - example.com # From listener 1.
+    - *.example.com # From listener 2 and 3.
+  secretName: example-com-tls
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: site-org-tls
+spec:
+  issuerRef:
+    name: my-issuer
+    kind: Issuer
+    group: cert-manager.io
+  dnsNames:
+    - site.org # From listener 4.
+  secretName: site-org-tls
+```
+
+## Supported Annotations
+
+If you are migrating to using Gateway resources instead of Ingress resources to
+generate Certificate resources, be aware that there are some differences in
+which annotations are [supported on the Ingress
+resource](https://cert-manager.io/docs/usage/ingress/#supported-annotations)
+versus which annotations are supported on the Gateway resource.
+
+The Gateway resource supports the following annotations:
+
+- `cert-manager.io/issuer`: the name of an `Issuer` to acquire the certificate
+  required for this Gateway. The Issuer _must_ be in the same namespace as the
+  Gateway resource.
+
+- `cert-manager.io/cluster-issuer`: the name of a `ClusterIssuer` to acquire the
+  certificate required for this Gateway. It does not matter which namespace
+  your Gateway resides, as `ClusterIssuers` are non-namespaced resources.
+
+- `cert-manager.io/issuer-kind`: the name of an external `Issuer`
+  controller's `CustomResourceDefinition` (only necessary for out-of-tree `Issuers`)
```

Comment:
This is wrong. It is the _Kind_ of the issuer resource type. `issuer-name` is the name of the resource.

`cert-manager.io/issuer=foo` is a short hand for `cert-manager.io/issuer-name=foo` `cert-manager.io/issuer-kind=Issuer`


Please add some wording around this distinction and that one is short hand for the other. This trips some users up when using external issuers.

Reply:
The description seems correct, albeit confusing. `issuer-kind` _is_ the name of an external Issuer's CRD.

What do you think about this:

```suggestion
- `cert-manager.io/issuer-kind`: the kind of the external issuer resource, for example `AWSPCACIssuer`. This is only necessary for out-of-tree issuers.
```